pkg/hardware/hardwarestructs.go

```diff
@@ -0,0 +1,61 @@
+package hardware
```

Comment:
Just a thought here. Since we are creating json based on `tinkerbell/tink` hardware type. We could probably use the upstream types?
[hardware](https://github.com/tinkerbell/tink/tree/main/protos/hardware)

Reply:
yep will update it to use upstream structs